plugin_runner/plugin_runner.py

```diff
@@ -708,7 +708,14 @@ def main(specified_plugin_paths: list[str] | None = None) -> None:
     port = "50051"
 
     executor = ThreadPoolExecutor(max_workers=settings.PLUGIN_RUNNER_MAX_WORKERS)
-    server = grpc.server(thread_pool=executor)
+    server = grpc.server(
+        thread_pool=executor,
+        options=(
+            # set max message lengths to 64mb
+            ("grpc.max_receive_message_length", 64 * 1024 * 1024),
+            ("grpc.max_send_message_length", 64 * 1024 * 1024),
```

Comment:
@beaugunderson do we need to pair this with changes in the gRPC client?

This stack overflow answer does, but unclear if it's required: https://stackoverflow.com/a/65009436